sound/soc/codecs/rt700.c

```diff
@@ -77,7 +77,7 @@ static int rt700_index_write(struct regmap *regmap,
 static int rt700_index_read(struct regmap *regmap,
 			    unsigned int reg, unsigned int *value)
 {
-	int ret;
+	int ret = 0;
```

Comment:
this init is not required, you have a ret = regmap_write(regmap, RT700_PRIV_DATA_R_H, val_h);

Reply:
Ok, got it.